GitUI/UserControls/RevisionGrid.cs

```diff
@@ -2321,13 +2321,12 @@ private void CreateTagToolStripMenuItemClick(object sender, EventArgs e)
                 return;
             }
 
-            using (var frm = new FormCreateTag(UICommands, LatestSelectedRevision))
-            {
-                if (frm.ShowDialog(this) == DialogResult.OK)
+            UICommands.DoActionOnRepo(() =>
                 {
-                    RefreshRevisions();
-                }
-            }
+                    var frm = new FormCreateTag(UICommands, LatestSelectedRevision);
```

Comment:
the form still needs to be disposed of

Reply:
Please compare to the create branch code that is a few lines below. I copied that to ensure code consistency with how previous contributors have written things.
Happy to make changes but I don't know what needs to change.

Reply:
The original code was `using (var frm = new FormCreateTag(UICommands, LatestSelectedRevision))`, it should be retained